ecommerce_extensions/management/commands/change_domain.py

```diff
@@ -0,0 +1,115 @@
+"""
+This module contains the command class to change
+the subdomain from prod domains to stage versions.
+"""
+import json
+import logging
+from urllib.parse import urlparse
+
+from django.contrib.sites.models import Site
+from django.core.management.base import BaseCommand
+
+LOGGER = logging.getLogger(__name__)
+CHANGE_DOMAIN_DEFAULT_SITE_NAME = 'stage.edunext.co'
+
+
+class Command(BaseCommand):
+    """
+    This class contains the methods to change
+    site prod domains to a stage version.
+    """
+    help = """This function will iterate over all Site objects
+            to change Site prod domains to a stage version."""
+    suffix_stage_domain = ""
+
+    def add_arguments(self, parser):
+        """
+        Add optional domain from the line command
+        """
+        parser.add_argument('suffix_domain', type=str,
+                            nargs='?', default=CHANGE_DOMAIN_DEFAULT_SITE_NAME,
+                            help='Suffix domain appended to ecommerce domains')
+        parser.add_argument('suffix_lms_domain', type=str,
+                            nargs='?', default=CHANGE_DOMAIN_DEFAULT_SITE_NAME,
+                            help='Suffix domain appended to LMS domains')
+
+    def handle(self, *args, **options):
+        """
+        This method will iterate over all Sites and SiteConfiguration
+        objects to change Site prod domains to a stage version.
+        """
+        self.suffix_stage_domain = options['suffix_domain']
```

Comment:
options will always have suffix_domain and suffix_lms_domain? or if one of them is missing this must fail?

Reply:
those options have a default value